src/GraphQl/Queries/ActionItemCategoryQueries.ts

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag';
```

Comment:
Why have you created a separate file just for this query ?

Reply:
Same as above...